.github/workflows/release.yml

```diff
@@ -10,39 +10,44 @@ jobs:
     runs-on: ubuntu-latest
     name: Start release process
     steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.BINARIES_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.BINARIES_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.BINARIES_AWS_REGION }}
       - name: Release
         id: release
+        uses: SonarSource/gh-action_release/main@v4
+        with:
+          publish_to_binaries: true
+          slack_channel: team-lang-phpy
```

Comment:
Can you please use `team-lang-phpy-notifs` instead? I know it was `team-lang-phpy` but I just realize it's not consistent with how we handle our other analyzers.